examples/plots/Four_Panel_Map.py

```diff
@@ -15,12 +15,12 @@
 
 ###########################################
 
-import netCDF4
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
-import numpy as np
+import netCDF4
```

Comment:
You realize it's still going to complain about netCDF4 being before matplotlib, right?